patterns/bottlerocket/main.tf

```diff
@@ -0,0 +1,48 @@
+################################################################################
+# Providers
+################################################################################
+data "aws_availability_zones" "available" {}
+
+data "aws_caller_identity" "current" {}
+
+locals {
+  name   = substr(basename(path.cwd), 0, 12)
+  region = "us-west-2"
+  tags = {
+    Sample     = local.name
+    GithubRepo = "github.com/aws-samples/amazon-eks-bottlerocket-mngnodegrp-terraform"
```

Comment:
lets be consistent with the other patterns https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/1ce59fc329c9bfbd0bdb68d0bc5736a3fb4988ca/patterns/karpenter/main.tf#L50-L53